eth_tester/backends/pyevm/main.py

```diff
@@ -419,18 +427,20 @@ def _max_available_gas(self):
         return header.gas_limit - header.gas_used
 
     def estimate_gas(self, transaction):
-        # TODO: move this to the VM level (and use binary search approach)
-        signed_evm_transaction = self._get_normalized_and_signed_evm_transaction(
+        evm_transaction = self._get_normalized_and_unsigned_evm_transaction(
```

Comment:
While pinned to the py-evm alpha 11, we should probably set `block_number='pending'` for gas estimates.

Reply:
I don't grok.  I thought all transactions are `pending`.  `block_number` isnt a transaction parameter. :space_invader:

Reply:
It's okay, I was misremembering something about how that worked. It only is used to change the nonce if it's not present. No change needed.